cli/resource/parse.go

```diff
@@ -78,6 +78,31 @@ func Parse(in io.Reader) ([]*types.Wrapper, error) {
 	return resources, err
 }
 
+// splitResources scans the content of the reader and splits the resources.
+// The resources should be separated by a line containing only "---".
```

Comment:
This reminds me that there is another bug in how yaml is handled here. The yaml specification allows comments on document separators, e.g. https://www.yaml.info/learn/document.html

If you could modify this slightly to split lines that _start_ with `---`, it would take care of that bug too.

Reply:
Added support for comments on the "---" line and updated unit test.